[TECH] Réusinage avancé de la partie "gestion d'une certification" de Pix Admin.

jbuget opened the pull request:
## :unicorn: Problème
> _Décrivez ici le besoin ou l'intention couvert par cette Pull Request._

## :robot: Solution
> _Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés._

## :rainbow: Remarques
> _Des infos supplémentaires, trucs et astuces ?_

## :100: Pour tester
> _Les instructions pour reproduire le problème, les profils de test, le parcours spécifique à utiliser, etc._

jbuget committed:
[Admin] Clean up component <CertificationDetailsAnswer/>

- move up template into /components folder
- glimmerize component
- replace emberx-select by ember-power-select component

The logic of this component and its whole bunch is still a big ball of mud.

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2059.review.pix.fr
- Orga: https://orga-pr2059.review.pix.fr
- Certif: https://certif-pr2059.review.pix.fr
- Admin: https://admin-pr2059.review.pix.fr
- API: https://api-pr2059.review.pix.fr/api/

 Please check it out!

jbuget committed:
[Admin] Clean up component <CertificationInfoField/>

- move up component template in the /components folder
- remove computed properties for style (padding)
- glimmerize component
- fix no-action template warning
- add tests

jbuget committed:
[Admin] Clean up component <CertificationInfoPublished/>

- move up template into /components folder
- glimmerize component
- rename argument "record" into "certification"

jbuget committed:
[Admin] Clean up component <CertificationDetailsCompetence/>

- move up template into /components folder
- glimmerize component

jbuget committed:
[Admin] Clean up component <CertificationInfoCompetence/>

jbuget committed:
[Admin] Clean up component <CertificationList/>

- move up template into /components folder

jbuget committed:
[Admin] Move 2 last component templates into /components folder

jbuget committed:
[Admin] Remove useless route /about

jbuget committed:
Fixup: certification-info-competences

jbuget committed:
[Admin] Clean up <CertificationStatusSelect> component

jbuget committed:
Fix regressions

jbuget committed:
Fix behaviours

jbuget committed:
Improve design

jbuget committed:
Au point où j'en suis…

jbuget committed:
Remove old computed property

jbuget committed:
Split code into components

jbuget committed:
Deep refactor of route certification/information

jbuget committed:
WIP

laura-bergoens commented:
A réouvrir lorsqu'on aura de la dispo pour le faire
